src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependencyNode.cs

```diff
@@ -266,7 +268,7 @@ public void SetProperties(string caption = null)
 
         public override int GetHashCode()
         {
-            return unchecked(Id.GetHashCode() + Resolved.GetHashCode());
+            return unchecked(Id.GetHashCode());
```

Comment:
ah i see, you mean we assume that Id does it correctly already. Yes, that's right we can skip it here.

---
In reply to: [99723696](https://github.com/dotnet/roslyn-project-system/pull/1469#discussion_r99723696) [](ancestors = 99723696)